Source/Manager.swift

```diff
@@ -70,15 +71,19 @@ public class Manager: NSURLProtocol {
         }
     }
 
+    static public func removeResource(resource: ResourceProtocol) -> ResourceProtocol? {
+        return resources.remove(resource.resourceIdentifier)
```

Comment:
I see now where the optional is coming from.  Does this function need to return a value at all (or if so, should it be `Bool` to indicate success)?